_includes/head.html

```diff
@@ -32,4 +32,9 @@
         <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
     <![endif]-->
 
+    <!-- Google Analytics -->
+    {% if site.google_analytics %}
```

Comment:
As discussed, I think that this suggestion is not blocking for the PR !

Reply:
I agree but then the build would have to be triggered with `JEKYLL_ENV=production bundle exec jekyll build` while we have no control over the jekyll build since we use github pages. Not really sure how to get around this :/

Reply:
Mind opening an issue for this @fe51 ? :pray: